.gitignore

```diff
@@ -77,3 +77,15 @@ deploy-key
 **/build/
 .classpath
 .project
+
+### VirtualEnv
+.Python
+[Bb]in
+[Ii]nclude
+[Ll]ib
+[Ll]ib64
+[Ll]ocal
+[Ss]cripts
```

Comment:
These ignores are dangerous as they are very general and have a relatively high chance for false positives. The ignores should point to the relevant locations (or be inside an appropriate directory like `/[Ll]ib`)

Reply:
If we change something here, we also need to change it in main